packages/ndla-ui/src/Pitch/Pitch.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
```

Comment:
Er tross alt ny fil.
```suggestion
 * Copyright (c) 2024-present, NDLA.
```